apps/socketoptions.hpp

```diff
@@ -216,6 +216,7 @@ const SocketOption srt_options [] {
     { "inputbw", 0, SRTO_INPUTBW, SocketOption::POST, SocketOption::INT64, nullptr},
     { "oheadbw", 0, SRTO_OHEADBW, SocketOption::POST, SocketOption::INT, nullptr},
     { "latency", 0, SRTO_LATENCY, SocketOption::PRE, SocketOption::INT, nullptr},
+    { "tsbpd", 0, SRTO_TSBPDMODE, SocketOption::PRE, SocketOption::BOOL, nullptr},
     { "tsbpddelay", 0, SRTO_TSBPDDELAY, SocketOption::PRE, SocketOption::INT, nullptr},
```

Comment:
As the option fixing happens - shouldn't this be removed, as an alias to `latency`?

Reply:
I saw you removed in somewhere in your PRs. So I left it as is. Would you like me to remove it in this PR?

Reply:
In PR #622 you delete `tsbpddelay`.